src/mono/mono-test.proj

```diff
@@ -0,0 +1,135 @@
+<Project>
+  <Import Project="Directory.Build.props" />
+  <Import Project="Directory.Build.targets" />
```

Comment:
Why the explicit imports here? The import of these will be automatic with MSBuild. 

Also the targets file should generally speaking come at the end of the file vs. the beginning.

Reply:
Don't know. I'm copying from `src/mono/mono.proj` (which probably copied it from `src/runtime/runtime.proj` which does it this way).

I tried taking out those two lines and I see my `<Message>`s printing:
```
  Creating .csproj files for mono runtime tests
  Mono obj dir is mono//
  Mono obj tests dir is mono//mono/tests/
  There are 2 tests, 2 simple
    Csproj mono/mono/tests/array-12193/array-12193.csproj
    Csproj mono/mono/tests/debug-casts/debug-casts.csproj
```
which is wrong (the obj dirs shuld be somewhere under `artifacts`).  So the `<Import>`s do _something_ and taking them out seems to break things.

Reply:
The reason why we import the `Directory.Build.targets` at the top is that we need to override the default Restore/Build/Pack targets. It was indeed copied from CoreCLR, not sure why the automatic import isn't working.